### ISSUE TITLE
[List] Allow a simple non-clickable List

### ISSUE THREAD

**Issue by stickfigure** [mlaursen/react-md#819]
The `List` is great but it seems overly specialized. Each item in the list always appears clickable (cursor + ink), unless the item is disabled, which greys the text. In neither case can you select text in a ListItem by click/drag.

I just want to present a list of non-clickable things that have avatars and/or icons.

Can the List be enhanced with some sort of mode that strips out the "this is clickable" behavior?

**Comment by mlaursen** [comment]
Yeah, I'm working on this for the v2 release here: [SimpleListItem](https://github.com/mlaursen/react-md/blob/next/packages/list/src/SimpleListItem.tsx)

But in the meantime, you can probably work around this by doing:

```jsx
const SimpleListItem = props => (
  <ListItem
    {...props}
    inkDisabled
    role={null}
    tabIndex={null}
    activeClassName=""
    activeBoxClassName=""
  />
);
```

Unfortunately it won't remove the event listeners on the `AccessibleFakeButton` in the list item, but it _should_ resemble a non-interactive item with that since it should no longer apply a hover class name, remove the button role, remove the tabIndex, and disable the ink effect.

**Comment by stickfigure** [comment]
This didn't seem to stop the active highlight but I overrode it (and the cursor) with CSS so I have something shippable now. I'll look forward to 2.0!  Thanks.

**Comment by startupgurukul** [comment]
Hi  Mikkel, 
Thanks for the great library, any ETA on v2.0

**Comment by mlaursen** [comment]
Hey @startupgurukul . Unfortunately... not yet :( I'm hoping to release the alpha within 2-3 weeks which will release all the packages in [this folder](https://github.com/mlaursen/react-md/tree/a52b2b274133c6736e3a2395516c1607a7a4551a/packages) minus the `forms`, `dev-utils`, and `documentation` but with a `layout` package (current similar component is `NavigationDrawer`).

I've been struggling getting "useful" documentation out of typescript, so generating my documentation site like I had before isn't really working which caused me to waste a lot of time trying different ways to generate it. So I'm going to try releasing these base packages (in alpha) without a documentation site and relying on documentation through IDEs since it's all with Typedoc right now. Once thats out there, I'll start working on the remaining packages and figure out how to create the documentation site from typedoc. I'll also update the #732 ticket with more info once I get closer to the first alpha release and some more detailed notes

**Comment by startupgurukul** [comment]
Looking forward.

**Comment by mlaursen** [comment]
This one _should_ be implemented now in v2 (which is at 2.0.0-alpha.1): https://react-md.dev/packages/list/demos#non-interactable-title

I'll keep this one open until the v2 release is closer and the new doc site is updated a bit more though.
